### PR TITLE
Return token set results.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -76,11 +76,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         }
 
         const {actor} = req.user;
-        await brAuthnToken.set({
+        const result = await brAuthnToken.set({
```
Comment (mattcollier) [inline lib/index.js:78]: I guess we're going to need `serviceId` here.

File: lib/index.js [hunk 2]
```diff
@@ -76,11 +76,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
...
           actor, account, email, type, hash, clientId,
           authenticationMethod, requiredAuthenticationMethods
         });
-        res.status(204).end();
+        res.json(result);
```
Comment (dlongley) [inline lib/index.js]: We can't do this for `nonce`, it returns the plain text challenge -- which would enable anyone to authenticate using a nonce token without having to access email/whatever notification service it should depend on.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: lib/index.js (hunk 2 above)
Comment (dlongley) [reply lib/index.js]: Oh, if github gives me more context here, I see we're not doing it for nonce. It is a bit weird to return 204 in one case and 200 in another. @davidlehn can we make this more clean somehow?

File: lib/index.js (hunk 1 above)
Comment (dlongley) [reply lib/index.js:78]: +1

**Review by dlongley — APPROVED** [review]
Note that we still need to be able to pass in `serviceId` per @mattcollier's comment.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**